### DISCUSSION TITLE
Qiitaにアクセスしづらい状態になっています

### DISCUSSION THREAD

**Discussion by kyntk** [increments/qiita-discussions#69] [障害情報]
現在Qiitaに大量のリクエストが送られており、アクセスしづらい状態となっております。
現在復旧作業中です。
ご利用の皆様にはご迷惑をおかけして申し訳ございません。
復旧し次第、再度こちらのアカウントにてお伝えします。

**Comment by kyntk** [comment]
抜本的な問題解決のための作業を引き続き行っています。
現在はアクセスが可能になっていますが、一部のページ等でレスポンスが遅延する可能性があります。

**Comment by ohakutsu** [comment]
Qiitaへの大量のリクエスト送信が発生しアクセスができない状態や一部のページでレスポンスの遅延等が発生していましたが復旧いたしました。
現在は問題なくご利用いただける状態となっております。ご迷惑をおかけし、申し訳ございませんでした。
